### DISCUSSION TITLE
Desktop dies with GPU error (6 beeps) when making AI subs.

### DISCUSSION THREAD

**Discussion by StickyChannel92** [SubtitleEdit/subtitleedit#8943] [Q&A]
I have been trying to make subs for a long time and when I do, my computer dies with 6 beeps on the motherboard, which indicates a GPU error. I tried everything, including reseating the GPU, update drivers, and update SubtitleEdit with no success. Sometimes, I might get a few subs but overall, it dies somewhere in the middle.

Subtitle Edit 4.0.8
Const-me model (also happens with the Purfview's Faster-Whisper)
Nvidia Geforce RTX 2060 Super
Intel Xeon e5 1620 v3
32GB RAM
Z440 Workstation

**Comment by realtechspecs** [comment]
> I have been trying to make subs for a long time and when I do, my computer dies with 6 beeps on the motherboard, which indicates a GPU error. I tried everything, including reseating the GPU, update drivers, and update SubtitleEdit with no success. Sometimes, I might get a few subs but overall, it dies somewhere in the middle.
> 
> Subtitle Edit 4.0.8 Const-me model (also happens with the Purfview's Faster-Whisper) Nvidia Geforce RTX 2060 Super Intel Xeon e5 1620 v3 32GB RAM Z440 Workstation

Were you trying to generate captions for a video or audio file when this happened?

**Reply by StickyChannel92** [reply]
Yes, I was. Btw, I did get a new GPU, which has never died on me ever. I found out that Purfview's faster whisper performs better than const-me in various cases. Some languages perform better on certain models than others.

- French performs better in Purfview's faster whisper, albeit with some corrections to be made.
- same with Portuguese.
- Galician seems to work just fine in const-me.
- can't even get whisperx to work for some reason.

Can cpp blas work on GPU and does it work better than the others?
